dojo/pipeline.py

```diff
@@ -88,10 +88,10 @@ def update_product_access(backend, uid, user=None, social=None, *args, **kwargs)
                     product.authorized_users.add(user)
                     product.save()
                 else:
-                    product_member, created = Product_Member.objects.get_or_create(product=product, user=user)
+                    product_member, created = Product_Member.objects.get_or_create(product=product, user=user, role=Role.objects.get(id=Roles.Owner))
```

Comment:
Now it will search for a product member with the owner role and create it if not present. I am not sure this is what we want here. If someone has changed the role to anything else for this product and user, it would be overriden. I think this would be better:

`product_member, created = Product_Member.objects.get_or_create(product=product, user=user, defaults={'role': Role.objects.get(id=Roles.Owner)})`